src/app/shell/personal-cabinet/parent/create-application/create-application.component.ts

```diff
@@ -68,8 +71,8 @@ export class CreateApplicationComponent implements OnInit, OnDestroy {
         this.store.dispatch(new GetAllUsersChildren());
       });
 
-    const workshopId = this.route.snapshot.paramMap.get('id');
-    this.store.dispatch(new GetWorkshopById(workshopId));
+    this.workshopId = this.route.snapshot.paramMap.get('id');
+    this.store.dispatch(new GetWorkshopById(this.workshopId));
```

Comment:
why do we need these changes?

Reply:
in case I pass to [routerLink] in html (on Cancel button) test runs forever (it doesn't fail but run for more than 200 sec)
Maybe there is another way to fix it that I don't know about